xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingAdapters.java

```diff
@@ -155,7 +185,7 @@ private static BigDecimal nextNodeAsDecimal(Iterator<JsonNode> iterator) {
     if (iterator == null || !iterator.hasNext()) {
       return null;
     }
-    return new BigDecimal(iterator.next().textValue()).stripTrailingZeros();
+    return new BigDecimal(iterator.next().textValue());
```

Comment:
why are we removing this striptrailingzeros?

Reply:
Removing the trailing zero's modifies the bigdecimal's internals, making it impossible to use them directly in the checksum code.

Reply:
it would be better to fix the check sum code then making it dependant from this line of code which can easly get changed in the future